src/infra/service/location.js

```diff
@@ -3,7 +3,9 @@ import { browserHistory } from 'react-router';
 import config from 'config';
 
 export const push = (to: string) => browserHistory.push(config.urlPrefix + to);
+export const urlPrefix = (to: string) => `${config.urlPrefix}${to}`
```

Comment:
@reyhansofian can you rename it to `getUrl` instead of `urlPrefix`? Just to make it more obvious. Copy and paste this:
```
export const getUrl = (to: string):string => `${config.urlPrefix}${to}`;
```
To make it type safe when imported.